### PR TITLE
removing outdated notebooks

### PR DESCRIPTION
**Pull request by atravitz**
The command line interface notebook is very outdated and I don't think it's a useful example anymore. I think  any additional CLI documentation should be added to the signac [Command Line Interface docs page](https://docs.signac.io/projects/core/en/latest/cli.html).

I believe advanced indexing is also outdated, unless this is just something that I don't personally use. I'd appreciate thoughts on if we should keep this, or if we should replace it with another topic.

### CODE REVIEW

**Comment by tommy-waltmann** [comment]
Is there information/examples on advanced indexing elsewhere in the documentation?

**Comment by atravitz** [comment]
> Is there information/examples on advanced indexing elsewhere in the documentation?

[Here](https://docs.signac.io/en/latest/indexing.html?highlight=indexing#concept) are our current docs for indexing. I'd like some feedback from folks on what they use it for and what type of docs would be helpful

**Comment by tommy-waltmann** [comment]
I haven't used the advanced indexing, feedback from others would be more useful here.

**Review by bdice — CHANGES_REQUESTED** [review]
We need to update `index.ipynb`, then this can be merged. We can re-organize the notebooks in a separate PR. I'm fine with leaving them numbered as they are (with nothing named "201") until we delete all the content we know we don't want.

**Comment by bdice** [comment]
@tommy-waltmann We discussed at signac dev meeting today -- basically all of the indexing-related APIs are deprecated (indexing can be somewhat useful for internal purposes but is not needed in the public API), so we will be reducing our discussion of indexing in docs and tutorials.

**Comment by bdice** [comment]
In @mikemhenry's review of #22, he suggested deleting notebooks 203 and 204 as well.

- 203 is definitely deprecated and can be removed.
- 204 is only marginally useful, if you need CLI wizardry or integration with a bash script. It says it is deprecated but I don't see the deprecated pieces. I think it's just not helpful.

@atravitz What do you think about deleting those two notebooks?

**Comment by atravitz** [comment]
> In @mikemhenry's review of #22, he suggested deleting notebooks 203 and 204 as well.
> 
> * 203 is definitely deprecated and can be removed.
> * 204 is only marginally useful, if you need CLI wizardry or integration with a bash script. It says it is deprecated but I don't see the deprecated pieces. I think it's just not helpful.
> 
> @atravitz What do you think about deleting those two notebooks?

I agree, I think 204 is very esoteric. We might want to add more examples of integrating with non-python workflows, but  I say remove this.

**Comment by atravitz** [comment]
Now that we've stripped out the unnecessary examples, it's looking a little sparse. 

image


I'd like people's thoughts on what else would be good content for examples. @bdice, @tommy-waltmann, @cbkerr, @kidrahahjo: what do you think about merging the [Advanced How-To](https://docs.signac.io/en/latest/recipes.html) section in with these examples to keep all these similar references in the same place? (at least as a temporary measure until we add more content)

**Comment by bdice** [comment]
@atravitz I think the Advanced How-To “recipes” section can become a set of notebooks, yes. But I would prioritize reorganizing the notebooks and merging the tutorial rST files with our existing notebooks first.

**Comment by cbkerr** [comment]
> what do you think about merging the Advanced How-To section in with these examples to keep all these similar references in the same place? 

@atravitz I think Examples and Tutorial sections are closer in scope than Examples and Advanced How To. Not all advice in Advanced How To fits as "Getting Started" material to someone first learning about the project.

> (at least as a temporary measure until we add more content)

I would caution against temporary moves unless it is showing up in both places at once.
For example, @AlainKadar mentioned losing something in the docs over the past months and I'd like to minimize that kind of thing happening.

**Comment by atravitz** [comment]
> > what do you think about merging the Advanced How-To section in with these examples to keep all these similar references in the same place?
> 
> @atravitz I think "Examples" and "Tutorial" sections are closer in scope than "Examples" and "Advanced How To". Not all advice in Advanced How To fits as "Getting Started" material to someone first learning about the project.
> 
> > (at least as a temporary measure until we add more content)
> 
> I would caution against temporary moves unless it is showing up in both places at once.
> For example, @AlainKadar mentioned losing something in the docs over the past months and I'd like to minimize that kind of thing happening.

In that case, I think the pandas example should be grouped with How-Tos. So
- remove pandas example here (or leave it as section 1.8)
- merge tutorials with examples
and
- make advanced how-tos notebooks
- add the pandas content to the how-tos

File: notebooks/index.ipynb [hunk 1]
```diff
@@ -19,19 +19,23 @@
     "* [1.5 Command Line Interface (CLI)](signac_105_Command_Line_Interface.ipynb)\n",
```
Comment (tcmoore3) [inline notebooks/index.ipynb:19]: This notebook was removed, right?

**Review by tcmoore3 — CHANGES_REQUESTED** [review]
Need to make sure the updated `index.ipynb` matches which files are actually removed.

**Review by bdice — APPROVED** [review]
I rewrote some notebooks using outdated FlowProject syntax. I'm going to merge this once CI passes -- this PR is a huge improvement over the current state of affairs.

**Pull request merged**